apogee/tools/__init__.py

```diff
@@ -23,6 +24,19 @@
                             if elem != 'ci' and elem != 'tiii')
     _ELEM_NUMBER_DICT['CI']= elements.__dict__['C'].number
     _ELEM_NUMBER_DICT['TiII']= elements.__dict__['Ti'].number
+
+# Detector limits used in pix2wv and wv2pix
+apStarRed_lo = 322
```

Comment:
I know it's going to be annoying to change everywhere, but can we give these the correct names? Pixel 322 is the low end of the blue detector.